tests/std/tests/Dev11_0836436_get_time/test.cpp

```diff
@@ -208,5 +250,297 @@ void test_990695() {
+    assert(read_date_locale(L"2020-\x0418\x044e\x041b\x044c-12", "ru_RU.UTF-8")
+           == make_tuple(12, /*NOTE DIFFERENCE:*/ 6, 120));
+
+    // Russian Auguest in different cases (short, long, mixed cases)
+    assert(read_date_locale(L"2020-\x0410\x0432\x0433\x0443\x0441\x0442-02", "ru_RU.UTF-8")
+           == make_tuple(2, /*NOTE DIFFERENCE:*/ 7, 120));
+    assert(
+        read_date_locale(L"2020-\x0430\x0432\x0433-02", "ru_RU.UTF-8") == make_tuple(2, /*NOTE DIFFERENCE:*/ 7, 120));
+    assert(read_date_locale(L"2020-\x0410\x0432\x0433\x0423\x0421\x0442-02", "ru_RU.UTF-8")
+           == make_tuple(2, /*NOTE DIFFERENCE:*/ 7, 120));
+
+    // Russian September in different cases (short, long, mixed cases)
+    assert(read_date_locale(L"2020-\x0421\x0435\x043d\x0442\x044f\x0431\x0440\x044c-21", "ru_RU.UTF-8")
+           == make_tuple(21, /*NOTE DIFFERENCE:*/ 8, 120));
+    assert(
+        read_date_locale(L"2020-\x0441\x0435\x043d-21", "ru_RU.UTF-8") == make_tuple(21, /*NOTE DIFFERENCE:*/ 8, 120));
+    assert(read_date_locale(L"2020-\x0421\x0435\x043d\x0442\x044f\x0411\x0440\x044c-21", "ru_RU.UTF-8")
+           == make_tuple(21, /*NOTE DIFFERENCE:*/ 8, 120));
+
+    // Russian October in different cases (short, long, mixed cases)
+    assert(read_date_locale(L"2020-\x041e\x043a\x0442\x044f\x0431\x0440\x044c-01", "ru_RU.UTF-8")
+           == make_tuple(1, /*NOTE DIFFERENCE:*/ 9, 120));
+    assert(
+        read_date_locale(L"2020-\x043e\x043a\x0442-01", "ru_RU.UTF-8") == make_tuple(1, /*NOTE DIFFERENCE:*/ 9, 120));
+    assert(read_date_locale(L"2020-\x041e\x043a\x0442\x044f\x0411\x0440\x044c-01", "ru_RU.UTF-8")
+           == make_tuple(1, /*NOTE DIFFERENCE:*/ 9, 120));
+
+    // Russian November in different cases (short, long, mixed cases)
+    assert(read_date_locale(L"2020-\x041d\x043e\x044f\x0431\x0440\x044c-09", "ru_RU.UTF-8")
+           == make_tuple(9, /*NOTE DIFFERENCE:*/ 10, 120));
+    assert(
+        read_date_locale(L"2020-\x043d\x043e\x044f-09", "ru_RU.UTF-8") == make_tuple(9, /*NOTE DIFFERENCE:*/ 10, 120));
+    assert(read_date_locale(L"2020-\x041d\x043e\x044f\x0411\x0440\x044c-09", "ru_RU.UTF-8")
+           == make_tuple(9, /*NOTE DIFFERENCE:*/ 10, 120));
+
+    // Russian December in different cases (short, long, mixed cases)
+    assert(read_date_locale(L"2020-\x0414\x0435\x043a\x0430\x0431\x0440\x044c-31", "ru_RU.UTF-8")
+           == make_tuple(31, /*NOTE DIFFERENCE:*/ 11, 120));
+    assert(
+        read_date_locale(L"2020-\x0434\x0435\x043a-31", "ru_RU.UTF-8") == make_tuple(31, /*NOTE DIFFERENCE:*/ 11, 120));
+    assert(read_date_locale(L"2020-\x0414\x0435\x043a\x0430\x0411\x0440\x044c-31", "ru_RU.UTF-8")
+           == make_tuple(31, /*NOTE DIFFERENCE:*/ 11, 120));
+}
+
+void test_locale_german() {
+    // German January in different cases (short, long, mixed cases)
+    assert(read_date_locale(L"2020-\x004a\x0061\x006e\x0075\x0061\x0072-05", "de_DE.utf-8")

```

Comment:
I didn't go through the German tests with the same level of scrutiny, but would like confirmation that these tests follow the same pattern of testing expanded version with "expected" casing, abbreviated in lowercase, and expanded version with mixed casing.

Reply:
> I didn't go through the German tests with the same level of scrutiny, but would like confirmation that these tests follow the same pattern of testing expanded version with "expected" casing, abbreviated in lowercase, and expanded version with mixed casing.

Yes, They do. Furthermore, I've added test cases to cover abbreviated versions with mixed casing. Although in both German and Russian locale the expanded and abbreviated versions of the month `May` seem to be Identical.